tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist =py27,py35
+[testenv]
+deps=
+    pytest
+    flexmock
+commands=py.test
```

Comment:
Change the command to `py.test {posargs}` to make it possible to pass arguments to py.test (for example `tox -- -x` would run `py.test -x`)